tox.ini

```diff
@@ -28,13 +28,15 @@ omit = tests/*
 show_missing = True
 
 [tox]
-envlist =
+env_list =
     checkqa
-    py{36,37,38,39}-dj{22,30,31,master}
+    py{38,39,310,311,312}-dj{22,30,31,32}
```

Comment:
I've updated tox to reflect that 3.6 - 3.8 are past 3.8 (3.8 recently, so I've left it in as pyenv still supports it, for now)

django `master` is now on 5.x, and actually this is broken by 4.x + due to changes in `Signal` args, so I removed it as well. 4+ is already disallowed in dependencies